x-pack/docs/en/security/authentication/remote-clusters-privileges.asciidoc

```diff
@@ -0,0 +1,296 @@
+[[remote-clusters-privileges]]
+=== Configure remote cluster privileges
```

Comment:
Nit: I feel this is better renamed to something like 
> Configure users and roles for remote cluster usage

because "remote cluster privileges" feels like it is a category of privilege such as "cluster privileges".

Reply:
I'll change this to **Configure roles and users for remote clusters**.